### PR TITLE
Local storage

### PR DESCRIPTION
**Pull request by Anand8317**
- If form data is submitted, it creates or updates in the local storage.
- If form data is locally stored, it fetches those values into the form.

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -141,12 +141,29 @@ popupClose.addEventListener('click', () => {
...
+      flag = 1;
     }
   }
+
+  if (flag === 0) {
+    localStorage.setItem('Name', text.value);
+    localStorage.setItem('Email', email.value);
+    localStorage.setItem('Message', mess.value);
+  }
 });
```
Comment (levy002) [inline js/index.js]: - [ ] Kindly make sure the form data for all inputs should be stored in a single object, and saved to a single local storage registry. 
    
     - ### Example
           const formData = {
           name: nameInput.value,
           email: emailInput.value,
           body: bodyInput.value,
           };

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @Anand8317  & @jaredatandi , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] Kindly set up linters correctly so it runs to check even the `es-lint` linter for JavaScript. You can use this[ setup](https://github.com/microverseinc/linters-config/tree/master/html-css-js).

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/index.js (hunk 1 above)
Comment (Anand8317) [reply js/index.js]: Okay, I have used stringify and parse to just use one object and one local storage registry. Thank you for your review.

File: js/index.js [hunk 2]
```diff
@@ -141,12 +141,33 @@ popupClose.addEventListener('click', () => {
...
   }
+
+  if (flag === 0) {
+    const formData = {
+      name: text.value,
+      email: email.value,
+      message: mess.value,
+    };
+    localStorage.setItem('data', JSON.stringify(formData));
+  }
```
Comment (mckent05) [inline js/index.js:165]: **OPTIONAL** - According to the project requirements, "After filling the form, and not submitting it, the entered values should be saved in the browser." Currently, your app doesn't function this way, I advise you to fix this issue 🙏. You can use the `input` event listener on your form to implement this functionality.

Your project:
- As you'd notice, I have filled the form, but the data is not preserved in the localstorage.
bndbjndb

**Review by mckent05 — APPROVED** [review]
Hi @Anand8317,

# STATUS: APPROVED ✔️ ✔️  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ 
- Your PR is descriptive and professional 👍 👍 
- Correct GitHub flow is used 👏 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**